demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1139,6 +1142,21 @@ def get_general_data(path: str, packs: Dict[str, Dict] = None):
     return {id_: data}
 
 
+def get_mp_v2_entities_data(path: str, packs: Dict[str, Dict] = None):
+    yaml_data = get_yaml(path)
+
+    id_ = yaml_data.get('id', '-')
+    name = yaml_data.get('name', '-')
+    fromversion = yaml_data.get('fromversion')
+    toversion = yaml_data.get('toVersion')
```

Comment:
Is it `toversion` or `toVersion` (lower or upper case `v`) ?

Reply:
I think the exact format and fields for those objects are not final yet (@ShahafBenYakir ?).
For now I defined this var just for passing it to the create_common_entity_data function.